<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuyProduct</class>
 <widget class="QMainWindow" name="BuyProduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>153</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>545</width>
    <height>153</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>545</width>
    <height>153</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_name">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Category - Manufacturer - Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="list_products"/>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_provider">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Provider</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_amount">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Amount</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox_amount"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="btn_buy">
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>41</height>
          </size>
         </property>
         <property name="text">
          <string>Buy</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
